Java Migration/Azure_DevOps_Server_2019.md

```diff
@@ -0,0 +1,41 @@
+
+
+Below diagram explains the steps to migrate from Oracle JRE to Azul Zulu OpenJDK for Azure DevOps Server 2019.
+
+![Java Migration flow](flow1.png)
+
+## Step 1: Pause Search indexing
+Go to https://github.com/Microsoft/Code-Search and find the right folder based on the TFS version you are using. For Azure DevOps Server 2019, go to https://github.com/Microsoft/Code-Search/tree/master/Azure_DevOps_Server_2019. To pause all indexing, execute the script PauseSearchIndexing.ps1 from Windows PowerShell with administrative privileges. You will be prompted to enter:
+
+* The SQL server instance name where the TFS configuration database resides.
+* The name of the TFS configuration database.
+
+## Step 2: Stop Elasticsearch Service
+Open Command Prompt as an administrator 
+
+If Elasticsearch is installed on the same server as TFS (local installation), use below command to locate ES folder. For remote search installations, locate the ES installation path and change the directory accordingly.
+### Change directory: 
+cd "C:\Program Files\Microsoft Team Foundation Server 15.0\Search\ES\elasticsearch-2.4.1\bin"
+### Stop the service:
+execute "elasticsearch-service.bat stop"
```

Comment:
@raggupta-ms : We should definitely cover that. But would it be better to capture that information as part of Pause/Resume Indexing jobs? so that the info is available for all scenarios

Reply:
Generally, these scripts are run during any customer reported issues. Means, there is already some issue going on and then existing job(s) might fail (there might already be jobs failing as there is some issue). So in that sense, that doesn't become a critical aspect to cover.
But here, customers would expect a seamless transition to zulu. Hence, it makes sense to inform them upfront.

Secondly, the only info we can convey with PauseIndexing (as a code comment may be?) is that any running jobs will keep running and any new coming jobs will get affected with this change.
It is upto the overall use case, such as in this case, where we are stopping ES later on, that will fail the job. So, it becomes a part of this documentation/blog to clearly convey that.

Reply:
For now, I see this as P2 issue. I will get this updated  in a day or 2. We shouldn't be gated on this to merge to Code Search repo